cms/djangoapps/contentstore/views/tests/test_organizations.py

```diff
@@ -1,15 +1,18 @@
 """Tests covering the Organizations listing on the Studio home."""
 import json
+import logging
 
-from django.urls import reverse
+from django.conf import settings
 from django.test import TestCase
+from django.urls import reverse
 from mock import patch
-
+from openedx.features.edly.tests.factories import EdlyOrganizationFactory, EdlySubOrganizationFactory, SiteFactory
```

Comment:
Google's Python style recommends:
`from openedx.features.edly.tests.factories import (
    EdlyOrganizationFactory,
    EdlySubOrganizationFactory,
    SiteFactory
)`
but it's your call we can live with it.

Reply:
@burhandodhy this feedback not addressed. Please not the indentation level.

Reply:
@burhandodhy this feedback is not addressed.